bin/waagent2.0

```diff
@@ -1117,7 +1117,148 @@ class oracleDistro(redhatDistro):
     def __init__(self):
         super(oracleDistro, self).__init__()
 
+############################################################
```

Comment:
The 2.0 library should not need these changes? Typically we do not update this code any more since this version of the agent is no longer supported.